### DISCUSSION TITLE
Implementation glacier module into mHM - current status

### DISCUSSION THREAD

**Discussion by ERottler** [mhm-ufz/mHM#6] [Q&A] [answered]
Hi!

We use mHM to simulate the Rhine River Basin and were wondering if there are **plans to implement a glacier module into mHM?** Maybe there are efforts undertaken in this direction already and a glacier module will be part of one of the next releases?

Erwin

**Comment by lese66** [comment, marked as answer]
Hi Erwin,

This is one of my top priorities. I had discussions with Bettina Schaefli but have not crystallize yet. You were working with a process based snow model, isn't it? This could be the start. Then we could include glacier mass evolution and melting. Do you have interest to work on it?

Luis

**Reply by ERottler** [reply]
Hi Luis!

Yes, we also work with a physically-based snow routine (from [ECHSE](http://echse.github.io/), which we recently implemented in [WASA-SED](https://github.com/TillF/WASA-SED)). The implementation of a physically-based snow routine and a glacier module might improve the mHM performance in mountainous areas, I think. Using MODIS snow cover maps in a multi-parameter calibration scheme and/or for model validation could be further steps. Yes, it sounds like a very cool task to figure out which snow/glacier routines fit to the mHM setup and to find a way to implement them. I would be very happy to help with this endeavor. If you want, we can meet online for a quick discussion the next days to see what is feasible.

Erwin
